deeppavlov/core/models/tf_model.py

```diff
@@ -125,7 +125,7 @@ def get_train_op(self,
                     variables_to_train.extend(tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES, scope=scope_name))
 
             if optimizer is None:
-                optimizer = tf.train.AdamOptimizer(learning_rate, **kwargs)
+                optimizer = tf.train.AdamOptimizer
```

Comment:
@mu-arkhipov Please confirm validity of the edit. (Seems valid to me)

Reply:
confermed